src/components/Client/ClientHistory.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const ClientHistory = () => {
+
+    return (
```

Comment:
remove return statement

Reply:
as I remove the return statement, I get the error:
ClientHistory(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.